docs/Concise-Guide-for-Developing-More-Secure-Software.md

```diff
@@ -0,0 +1,29 @@
+12. **Make it easy for your users to update**. Implement stable APIs, e.g., support old names when new ones are added. Use semantic versioning. Have a deprecation process.
+13. **Sign your project's important releases**. Use standard tools and signing formats for your distribution. See the [cosign tool](https://docs.sigstore.dev/cosign/overview) from the [sigstore project](https://www.sigstore.dev/) to sign containers and other artifacts.
+14. [**Earn an OpenSSF Best Practices badge**](https://bestpractices.coreinfrastructure.org/) for your open source project. At least earn “passing”. Plan and roadmap to eventually earn silver & gold.
+15. **Improve your** [**OpenSSF Scorecards**](https://github.com/ossf/scorecard) **score (if OSS and on GitHub)**. You can read the [Scorecards checks](https://github.com/ossf/scorecard#scorecard-checks). Use the [Allstar](https://github.com/ossf/allstar) monitor.
+16. **Notify the community of vulnerabilities in your project.** Publish security advisories with accurate & precise information, e.g., what usage & versions are vulnerable, mitigations, and fixed version(s). Get a CVE ID. On GitHub, [create your security advisory](https://docs.github.com/en/code-security/repository-security-advisories/creating-a-repository-security-advisory#creating-a-security-advisory) & [request a CVE](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories#cve-identification-numbers).
+17. **Improve your** [**Supply chain Levels for Software Artifacts (SLSA)**](https://slsa.dev/) **level**. This hardens the integrity of your build and distribution process against attacks.
+18. **Publish and consume a software bill of materials (SBOM)**. This lets users verify inventory, id known vulnerabilities, & id potential legal issues. Consider [**SPDX**](https://spdx.dev/) or [**CycloneDX**](https://cyclonedx.org/).
+19. **Onboard your project into** [**LFX Security**](https://security.lfx.linuxfoundation.org/) **if you manage a Linux Foundation project**.
+20. **Apply the** [**CNCF Security TAG Software Supply Chain Best Practices guide**](https://github.com/cncf/tag-security/blob/main/supply-chain-security/supply-chain-security-paper/CNCF_SSCP_v1.pdf).
+21. **Implement** [**ASVS**](https://owasp.org/www-project-application-security-verification-standard/) **and follow relevant** [**cheatsheets**](https://cheatsheetseries.owasp.org/index.html).
+22. **Apply SAFECode's** [**Fundamental Practices for Secure Software Development**](https://safecode.org/uncategorized/fundamental-practices-secure-software-development/).
+23. **Complete a third-party security code review/audit**. Expect this to be USD$50K or more.
```

Comment:
```suggestion
23. **Complete a third-party security code review/audit**. Expect this to be $50,000 USD or more.
```

@david-a-wheeler I also noticed we kept `USD$50K` in 23. I think being slightly more explicit here would be beneficial as what we have right now, might not be as intuitive, at first glance.

Reply:
Okay. People might want it shortened, but the meaning is unchanged.